contrib/inventory/spacewalk.py

```diff
@@ -57,12 +57,15 @@
     import simplejson as json
 
 base_dir  = os.path.dirname(os.path.realpath(__file__))
+default_ini_file = os.path.join(base_dir, "spacewalk.ini")
+
 SW_REPORT = '/usr/bin/spacewalk-report'
 CACHE_DIR = os.path.join(base_dir, ".spacewalk_reports")
 CACHE_AGE = 300 # 5min
-INI_FILE = os.path.join(base_dir, "spacewalk.ini")
+INI_FILE = os.path.expanduser(os.path.expandvars(os.environ.get("SPACEWALK_INI", default_ini_file)))
```

Comment:
I think `SPACEWALK_INI_PATH` would be the most consistent. I'll make that change.